doc/classes/Object.xml

```diff
@@ -241,16 +241,12 @@
 				public override void _Ready()
 				{
 				    var button = new Button();
-				    // Option 1: Object.Connect() with an implicit Callable for the defined function.
-				    button.Connect("button_down", OnButtonDown);
-				    // Option 2: Object.connect() with a constructed Callable using a target object and method name.
-				    button.Connect("button_down", new Callable(self, nameof(OnButtonDown)));
-				    // Option 3: Signal.connect() with an implicit Callable for the defined function.
-				    button.ButtonDown.Connect(OnButtonDown);
-				    // Option 3b: In C#, we can use signals as events and connect with this more idiomatic syntax:
+				    // Option 1: In C#, we can use signals as events and connect with this idiomatic syntax:
 				    button.ButtonDown += OnButtonDown;
-				    // Option 4: Signal.connect() with a constructed Callable using a target object and method name.
-				    button.ButtonDown.Connect(new Callable(self, nameof(OnButtonDown)));
+				    // Option 2: Object.Connect() with an implicit Callable for the defined function.
+				    button.Connect("button_down", OnButtonDown);
+				    // Option 3: Object.Connect() with a constructed Callable using a target object and method name.
+				    button.Connect("button_down", new Callable(this, nameof(OnButtonDown)));
```

Comment:
can do this now
`button.Connect(Button.SignalName.ButtonDown, new Callable(this, MethodName.OnButtonDown));`